app/frontend/src/components/admin/AdminReviewSubmission.vue

```diff
@@ -0,0 +1,169 @@
+<template>
```

Comment:
this all looks great. there were a few changes to this template and css in pr-78 to improve layout if we want them.

Reply:
nevermind.. i just had a better look at the ui for this pr and it looks great